packages/angular/ssr/test/app_spec.ts

```diff
@@ -26,14 +27,32 @@ describe('AngularServerApp', () => {
     })
     class HomeComponent {}
 
+    @Component({
+      selector: 'app-redirect',
+    })
+    class RedirectComponent {
+      constructor() {
+        const router = inject(Router);
+        const pendingTasks = inject(PendingTasks);
+
+        pendingTasks.run(() =>
+          router.navigate([], {
```

Comment:
Can we maybe do this in a guard instead? As discussed, I don’t love that this causes the navigation to not complete successfully.